.github/workflows/erlang.yml

```
@@ -16,8 +13,10 @@ jobs:
       image: erlang:22.3.3
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Compile
-      run: rebar3 compile
-    - name: Run tests
-      run: rebar3 do xref, dialyzer, eunit
+      - uses: actions/checkout@v2
+      - name: Compile
+        run: ./rebar3 compile
+      - name: Run xref and dialyzer
+        run: rebar3 do xref, dialyzer
+      - name: Run eunit
+        run: rebar3 as gha do eunit
```

```suggestion
        run: ./rebar3 as gha do eunit
```

I assume you want to run the rebar version supplied not to get surprised by incompatibilities in rebar?

---

Done